taichi/rhi/opengl/CMakeLists.txt

```diff
@@ -24,3 +24,4 @@ if(NOT ANDROID)
 target_link_libraries(opengl_rhi PRIVATE glfw)
 endif()
 target_link_libraries(opengl_rhi PRIVATE spirv-cross-glsl spirv-cross-core)
+target_link_libraries(opengl_rhi PUBLIC common_rhi)
```

Comment:
ooc, why is this `PUBLIC`? Isn't `common_rhi` always statically linked

Reply:
Oh it's a typo here, public linking `common_rhi` with `ti_device_api` should be enough